src/matrixroomhandler.ts

```diff
@@ -266,11 +266,19 @@ export class MatrixRoomHandler {
             initial_state: [
                 {
                     content: {
-                        join_rule: "public",
+                        // TODO: We only really support "public", since we don't know who to invite.
```

Comment:
this is rather.....meh

Reply:
Not much we can do:
 - We don't know who is trying to join
 - We can't yet map matrix users to discord users to determine who is allowed. We also can't "whitelist users".

Reply:
It might be better to not have that option at all, then?

Reply:
that or a script for the AS bot to invite someone in conjunction with this

Reply:
>invite someone

the problem is who :). But I agree, we could probably remove this option since it's useless.

Reply:
How is it a problem? Something like adminme but instead it invites someone, shouldn't be far-fetched

Reply:
Seems like more of a job for provisioning than aliases though.

Reply:
what do you mean? An `npm run inviteme` script would fix the issue for now, would it not?

Reply:
So what I would do in this instance is write a script that provisions an invite only room and invites the user who invoked it. Configuring your bridge to do invite only rooms on alias requests is weird because it's always going to access deny you.

The *other* option is configuring the bridge to always invite the same user(s) when creating these rooms, which might work for self hosted instances.

Reply:
> invites the user who invoked it

How do we know who revoked an alias query?

Reply:
We don't know who *in*voked the alias :(